docs/schema/2.0/manifest.schema.json

```diff
@@ -359,6 +361,9 @@
               "component": {
                 "type": "string"
               },
+              "filter": {
```

Comment:
It does not, I just wanted to put it next to the url. To be honest I do not understand why the manigfest has this host node whatsoever.

Reply:
Let's put it one lever higher and rename it to
```suggestion
              "mailingLinksUrlFilter": {
```

Reply:
No problem to move it up. But do we really want to rename it? All the other attributes are much less specific and some are a bit multipurpose. We might want a filter for another extension in the future, wdyt?

Reply:
I'm also still considering using `oneOf` (see https://stackoverflow.com/a/25033301) to show what is required for which extension type.

Reply:
IMO using specific field is approriate here, because there is no other extension using "filter" that we need. I'd use `filter` only if we have a plan for more/all extensions to leverage it in certain way, which enhances the flexibility of the schema. Is there any other extension that would use it?

Using `oneOf` is a good move, we need that on the `extension` level. For now it'd be just two options, 1. what we have today as a common ground for backwards compatiility, 2. the new object type for mailing links with the added field (in that case it could be named `filter`. We could make additions to the oneOf every time we will need to extend particular extension schema with a new field.